src/components/CompleteProfile/CompleteProfile.css

```
@@ -119,7 +128,61 @@
 			.slider-checkbox-row {
 				display: grid;
 				gap: var(--spacing-xs);
-				grid-template-columns: 1fr 8fr;
+				grid-template-columns: repeat(1fr, 2fr);
```

I think there is a typo here... The `repeat` will be ignored.

---

There is still a typo here...

```suggestion
				grid-template-columns: repeat(2, 1fr);
```

See: https://developer.mozilla.org/en-US/docs/Web/CSS/repeat